.github/workflows/main.yml

```diff
@@ -1,9 +1,14 @@
 name: CI
 on:
   push:
-    branches-ignore: [master]
+    branches:
+      - auto-cargo
+      - try
+      - automation/bors/try
+      - "rust-**"
```

Comment:
Why add rust-** here?

Reply:
CI is currently run on those branches and I didn't wanted to change that.

Reply:
I don't think it is needed. Those are beta branches, and those go through bors like a normal `master` merge (and thus are tested on the `auto-cargo` branch).

Reply:
Sure, but don't you want the CI checks to be run before doing `@bors r+` ?

Otherwise without this rule I don't think there will be any CI checks when targeting the beta branches.

Reply:
Shouldn't that be handled by the `pull_request` trigger?

Reply:
Oh! You're probably right, I thought we were in the `pull_request` section, not the `push` one.

Reply:
Removed.